package.json

```diff
@@ -8,7 +8,7 @@
     "build": "babel src/ --out-dir lib/ --copy-files --ignore ___tests___ --source-maps",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "license": "license-checker --onlyAllow 'Apache-2.0; BSD; BSD-2-Clause; BSD-3-Clause; ISC; MIT; Unlicense; WTFPL; CC-BY-3.0; CC0-1.0' --production",
-    "lint": "eslint index.js && eclint check $(git ls-files) && markdownlint README.md",
+    "lint": "eslint src/index.js && eclint check $(git ls-files) && markdownlint README.md",
```

Comment:
could you eslint src/* here?

Reply:
Yep, that was something I realized yesterday, that the original code probably only had a single file. I will check, most probably that will mean more changes in the other files and the flow plugin I mentioned earlier, because it's now flow-syntax.

Reply:
I'm adding in a minute a new PR for the eslint / flow support, the amount of changes needed was significant.

Reply:
Does the other PR build on top of this one, so I can merge this already. ok?

Reply:
Yes, the other PR builds on top of this one, no merge issues should happen